assets/src/modules/config/BaseLayer.js

```diff
@@ -924,6 +924,19 @@ export class BaseLayersConfig {
             }
         }
 
+        // Add base layer as project default background color
+        const default_background_color_index = options.hasOwnProperty('default_background_color_index') ? options.default_background_color_index : -1;
+        if (isNaN(default_background_color_index)) {
+            throw new ValidationError('The default_background_color_index parameter is not a number!');
+        }
+        if (default_background_color_index !== -1) {
```

Comment:
Linked to @Gustry comment
```suggestion
        if (default_background_color_index !== -1
            && names.indexOf('empty') == -1
            && names.indexOf('project-background-color') == -1) {
```